Jenkinsfile.conda

```diff
@@ -1,76 +1,2 @@
-    agent { label 'CSC_Conda_Node'}
-
-    environment {
-        package_name = "ddsconfig"
-        dockerImageName = "lsstts/conda_package_builder:latest"
-        container_name = "salobj_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
-    }
-    stages {
-        stage("Pull Docker Image") {
-            steps {
-                script {
-                sh """
-                docker pull ${dockerImageName}
-                """
-                }
-            }
-        }
-        stage("Start builder"){
-            steps {
-                script {
-                    sh """
-                    docker run --name ${container_name} -di --rm \
-                        -v ${WORKSPACE}:/home/saluser/source ${dockerImageName}
-                    """
-                }
-            }
-        }
-        stage("Create Conda package") {
-            steps {
-                script {
-                    sh """
-                    docker exec ${container_name} sh -c 'cd ~/source/conda && source ~/miniconda3/bin/activate && source "\$OSPL_HOME/release.com" && conda build --prefix-length 100 .'
-                    """
-                }
-            }
-        }
-        stage("Push Conda package") {
-            when { expression { env.TAG_NAME != null } }
-            steps {
-                withCredentials([usernamePassword(credentialsId: 'CondaForge', passwordVariable: 'anaconda_pass', usernameVariable: 'anaconda_user')]) {
-                    script {
-                        sh """
-                        docker exec ${container_name} sh -c "source ~/miniconda3/bin/activate && \
-                            anaconda login --user ${anaconda_user} --password ${anaconda_pass} && \
-                            anaconda upload -u lsstts --force \
-                            ~/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2"
-                        """
-                    }
-                }
-            }
-        }
-        stage("Push develop conda package") {
-            when { expression { env.TAG_NAME == null } }
-            steps {
-                withCredentials([usernamePassword(credentialsId: 'CondaForge', passwordVariable: 'anaconda_pass', usernameVariable: 'anaconda_user')]) {
-                    script {
-                        sh """
-                        docker exec ${container_name} sh -c "source ~/miniconda3/bin/activate && \
-                            anaconda login --user ${anaconda_user} --password ${anaconda_pass} && \
-                            anaconda upload -u lsstts --force  --label dev \
-                            ~/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2"
-                        """
-                    }
-                }
-            }
-        }
-    }
-    post {
-        cleanup {
-            sh """
-            docker stop ${container_name}
-            """
-        }
-    }
-}
+@Library('JenkinsShared')_
+CondaPipeline("", "ts-ddsconfig", "lsst.ts.ddsconfig")
```

Comment:
Please use [] instead of "" for the config packages, for consistency (easier to understand if it's *always* a list).